HIGHT : 3  #初始飞行高度 单位：m
vel_z : 0.1  #z轴速度
Kp : 0.8 #速度系数，不宜太大
Ki : 0.0002
Kd : 1.088 
MIN_ERROR : 0.20 #可接受误差
vel_lit : 0.3 #速度限制
x_move : 3 # 起飞后搜索的距离 单位：m 
y_move : 5


channle1 :  #升降舵通道值
 value0 : 999  #最大值 最前方
 value1 : 1065  #中间值
 value2 : 1933 #最小值
channle2 :  #方向舵通道值
 value3 : 1065  #最左侧值
 value4 : 1499 #中间值
 value5 : 1933 #最右侧值

cam: #相机参数
 x_err : 0 #x轴相对误差 单位：cm
 y_err : 0 #y轴相对误差
 R: 90 #相机旋转角度,以相机坐标系x正与机头一致为0度，顺时针为正，逆时针为负